drivers/scheduler/k8s/specs/fio-low-io/fio-low-io.yaml

```diff
@@ -0,0 +1,66 @@
+apiVersion: apps/v1
+kind: StatefulSet
+metadata:
+  name: fio-low-io
+spec:
+  serviceName: fio-low-io
+  {{ if .Replicas }}
+  replicas: {{ .Replicas }}
+  {{ else }}
+  replicas: 1
+  {{ end }}
+  selector:
+    matchLabels:
+      app: fio
+  template:
+    metadata:
+      labels:
+        app: fio
+    spec:
+      schedulerName: stork
+      containers:
+      - name: fio
+        # image: portworx/fio_drv
+        image: alicelol/fio_drv:alyu-test
```

Comment:
can we push this image to portworx repo only with fio version name as tag?

Reply:
I think that is a great point. However I don't have permission to access the portworx docker registry. Can you help? @lsrinivas-pure